ethcore/wasm/src/runtime.rs

```diff
@@ -567,6 +568,28 @@ impl<'a, 'b> Runtime<'a, 'b> {
 		&*self.memory
 	}
 
+	fn mem_cmp(&mut self, context: InterpreterCallerContext)
+		-> Result<Option<interpreter::RuntimeValue>, InterpreterError>
+	{
+		//
+		// method signature:
+		//   fn memcmp(cx: *const c_void, ct: *const c_void, n: usize) -> i32;
```

Comment:
https://github.com/paritytech/pwasm-std/blob/master/libc/src/lib.rs#L8

should match here